src/adapters/aave-v3/products/a-token/aaveV3ATokenAdapter.ts

```diff
@@ -52,13 +50,7 @@ export class AaveV3ATokenPoolAdapter extends AaveBasePoolAdapter {
     return reserveData.liquidityRate
   }
```

Comment:
the templates import get transaction params from `import { GetTransactionParams } from '../../..'`

Should we keep consistent?

Reply:
Yes. There are advantages of doing it that way. I changed it for the already implemented ones and forgot to revert.

Reply:
Actually. I'll have to change the templates, as the name clashes.